### PR TITLE
Align docs to new test suites user flow

### PR DESCRIPTION
**Pull request by jonbell-lot23**
1. Added npx replayio install step to docs, because it's in the app
2. Aligned to npm install --save-dev @replayio/playwright rather than npm install @replayio/playwright -D

### CODE REVIEW

**Review by filiphric — APPROVED** [review]

File: src/app/basics/getting-started/record-your-playwright-tests/page.md [hunk 1]
```diff
@@ -22,33 +22,39 @@ It will create an API key and guide you through each step.
...
-pnpm install @replayio/playwright -D
+pnpm install --save-dev @replayio/playwright
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-bun install @replayio/playwright -D
+bun install --save-dev @replayio/playwright
```
Comment (jasonLaster) [inline src/app/basics/getting-started/record-your-playwright-tests/page.md:46]: what's wrong with `bun install -D @replayio/playwright` and others as well?

Comment (jonbell-lot23) [reply src/app/basics/getting-started/record-your-playwright-tests/page.md:46]: Docs had this syntax:
npm install @replayio/playwright -D

Our orange new team flow had this syntax:
npm install --save-dev @replayio/playwright

So this change is aligning with the more recent and baked-in style we see in the app. I also think there's benefit to writing flags out to make them easier to read and understand, but that's a subjective copy thing.

Comment (jasonLaster) [reply src/app/basics/getting-started/record-your-playwright-tests/page.md:46]: That's fine.

**Review by jasonLaster — APPROVED** [review]

File: src/app/basics/getting-started/record-your-playwright-tests/page.md (hunk 1 above)
Comment (bvaughn) [reply src/app/basics/getting-started/record-your-playwright-tests/page.md:46]: I think Bun should be
```suggestion
bun add --dev @replayio/playwright
```

File: src/app/basics/getting-started/record-your-playwright-tests/page.md [hunk 2]
```diff
@@ -22,33 +22,39 @@ It will create an API key and guide you through each step.
...
-yarn add @replayio/playwright -D
+yarn add --save-dev @replayio/playwright
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-pnpm install @replayio/playwright -D
+pnpm install --save-dev @replayio/playwright
```
Comment (bvaughn) [inline src/app/basics/getting-started/record-your-playwright-tests/page.md:39]:
```suggestion
pnpm add --save-dev @replayio/playwright
```

File: src/app/basics/getting-started/record-your-playwright-tests/page.md [hunk 3]
```diff
@@ -22,33 +22,39 @@ It will create an API key and guide you through each step.
...
-npm install @replayio/playwright -D
+npm install --save-dev @replayio/playwright
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-yarn add @replayio/playwright -D
+yarn add --save-dev @replayio/playwright
```
Comment (bvaughn) [inline src/app/basics/getting-started/record-your-playwright-tests/page.md:32]:
```suggestion
yarn add --dev @replayio/playwright
```

File: src/app/basics/getting-started/record-your-cypress-tests/page.md [hunk 4]
```diff
@@ -22,21 +22,21 @@ It will create an API key and guide you through each step.
...
-npm install @replayio/cypress -D
+npm install --save-dev @replayio/cypress
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-yarn add @replayio/cypress -D
+yarn add --save-dev @replayio/cypress
```
Comment (bvaughn) [inline src/app/basics/getting-started/record-your-cypress-tests/page.md:32]:
```suggestion
yarn add --dev @replayio/cypress
```

File: src/app/basics/getting-started/record-your-cypress-tests/page.md [hunk 5]
```diff
@@ -22,21 +22,21 @@ It will create an API key and guide you through each step.
...
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-pnpm install @replayio/cypress -D
+pnpm install --save-dev @replayio/cypress
```
Comment (bvaughn) [inline src/app/basics/getting-started/record-your-cypress-tests/page.md:39]:
```suggestion
pnpm add --save-dev @replayio/cypress
```

**Review by bvaughn — COMMENTED** [review]
I think the syntax/format should be:

```sh
bun add --dev <packge>
npm install --save-dev <package>
pnpm add --save-dev <package>
yarn add --dev <packge>
```

**Comment by jonbell-lot23** [comment]
Thanks!

File: src/app/basics/getting-started/record-your-cypress-tests/page.md [hunk 6]
```diff
@@ -49,6 +49,12 @@ bun install @replayio/cypress -D
 {% /tab %}
```
Comment (bvaughn) [inline src/app/basics/getting-started/record-your-cypress-tests/page.md:49]: Looks like Bun got missed in the Cypress page, cc @jonbell-lot23

Comment (jonbell-lot23) [reply src/app/basics/getting-started/record-your-cypress-tests/page.md:49]: Thanks, fixed here: https://github.com/replayio/docs/pull/190

**Pull request merged**